packages/node/src/sdk.ts

```diff
@@ -179,6 +180,17 @@ export function init(options: NodeOptions = {}): void {
   }
 
   updateScopeFromEnvVariables();
+
+  if (options.spotlight) {
+    const client = getCurrentHub().getClient();
+    if (client && client.addIntegration) {
```

Comment:
Can we check if this _has_ run, and only then call this? Even if we check this inside of `setupIntegrations()`, may be a bit cleaner I'd say?

Reply:
Discussed this offline quickly: Checking this is not easy outside the `BaseClient` class. Given this call no-ops if integrations were already set up, let's keep it as is for now.